brevbaker-api-model-common/src/main/kotlin/no/nav/pensjon/brev/api/model/maler/LetterRequest.kt

```diff
@@ -15,6 +15,16 @@ data class BestillBrevRequest<T : Brevkode<T>>(
     val language: LanguageCode,
 ) : BrevRequest<T>
 
+@Suppress("unused")
+data class BestillBrevRequestAsync<T : Brevkode<T>>(
```

Comment:
Litt usikker på korfor vi ikkje har gjort om desse også til class med internal constructors, men det burde dei jo vera.

Reply:
Kan ta ein separat runde på det, det er jo eigentleg utanfor skop for denne PR-en.